src/components/Alerts.test.js

```diff
@@ -0,0 +1,82 @@
+import '@testing-library/jest-dom'
+
+import { fireEvent, getAllByTestId, getByText, render } from '@testing-library/react'
+import _ from 'lodash/fp'
+import { h } from 'react-hyperscript-helpers'
+import Alerts from 'src/components/Alerts'
+import { useServiceAlerts } from 'src/libs/service-alerts'
+import * as Utils from 'src/libs/utils'
+
+
+jest.mock('src/libs/service-alerts', () => {
+  const originalModule = jest.requireActual('src/libs/service-alerts')
+  return {
+    ...originalModule,
+    useServiceAlerts: jest.fn()
+  }
+})
+
+beforeAll(() => {
+  const modalRoot = document.createElement('div')
+  modalRoot.id = 'modal-root'
+  document.body.append(modalRoot)
+})
+
+afterAll(() => {
+  document.getElementById('modal-root').remove()
+})
+
+const testAlerts = [
+  {
+    id: 'abc',
+    title: 'The systems are down!',
+    message: 'Something is terribly wrong'
+  },
+  {
+    id: 'def',
+    title: 'Scheduled maintenance',
+    message: 'Offline tomorrow'
+  }
+]
+
+describe('Alerts', () => {
```

Comment:
Worth checking for the "no alerts" case?

Reply:
Added in https://github.com/DataBiosphere/terra-ui/pull/3361/commits/6248e02da0d1448ff518f63ca99b9a05982a572b.